pkg/urlutil/git.go

```diff
@@ -8,11 +11,13 @@ var (
 		"github.com/",
 		"git@",
 	}
+
+	urlPathWithFragmentSuffix = regexp.MustCompile(".git(:?#.+)?$")
```

Comment:
gosh, sorry, you're right. I meant `?:`. I've fixed the code and added tests for both cases.